### ISSUE TITLE
feature request: leaderboard

### ISSUE THREAD

**Issue by jonmorehouse** [benmorehouse/rtc#4]
I was thinking about this today, and I'd really like some type of leaderboard functionality for my own usage.

For instance, let's say I'd like to work on my consistency. I want to log 5 hours in my terminal application every Monday through Friday. Keeping a leaderboard of days in a row to gamify it (just against myself) would be really cool. 

Even better if it can update in slack when I'm coming close to a record.

What do you think? Not sure if it's in scope of this project, but it would be pretty neat!

**Comment by benmorehouse** [comment]
Yeah i am definitely a fan of this sort of enhancement. I see a few possible ways of taking care of this.

Firstly, I like the integration into slack for this project. That would be more useful as a way to actually provide feedback to the user on where they are at vs only being able to access the data when they trigger the cli.

Secondly, I see where you are coming from with the dashboard. My initial thoughts are to expand on this project more: at the moment, it only wraps the rescuetime api enough to get daily feedback. 

Marking this issue as a possible enhancement to the project. More to come soon. @jonmorehouse
